bitrise.yml

```diff
@@ -334,6 +336,69 @@ workflows:
       bitrise.io:
         stack: osx-xcode-15.2.x
         machine_type_id: g2-m1.8core
+  determine_apps_affected:
+    steps:
+    - git-clone@6.2: {}
```

Comment:
If you add it here, you can remove it from the workflow (1_git_clone_and_post_clone) below, step in line 398.

Reply:
Forget my previous comment after what we have talked